src/puppetlabs/puppetdb/mq.clj

```diff
@@ -10,7 +10,22 @@
             [puppetlabs.puppetdb.utils :as utils]
             [puppetlabs.puppetdb.schema :refer [defn-validated]]
             [schema.core :as s]
-            [slingshot.slingshot :refer [throw+]]))
+            [slingshot.slingshot :refer [throw+]]
+            [metrics.timers :refer [timer time!]]
+            [puppetlabs.puppetdb.metrics.core :as metrics]))
+
+(def mq-metrics-registry (get-in metrics/metrics-registries [:mq :registry]))
+
+(defn keyword->metric-name
```

Comment:
I might move this to `puppetlabs.puppetdb.metrics.core` since I can think of a few other places we might wanna use it :)